packages/@ember/-internals/glimmer/tests/integration/syntax/each-in-test.js

```
@@ -672,6 +672,44 @@ moduleFor(
       </ul>
     `);
     }
+
+    [`@test it supports native ES6 Sets`]() {
```

Hmm, why use `{{#each-in` with a `Set`? I would have expected to use `{{#each` 🤔

---

Agreed, but this behavior did exist beforehand. We could throw an error instead potentially, not sure if it was considered public API.

---

Throwing seems better to me, also I assume this is just something you noticed in passing (since there are no linked issues)?

---

Yeah. Ok, sounds good 👍